migrations/inputs_rabbitmq/migration.go

```diff
@@ -0,0 +1,66 @@
+package inputs_rabbitmq
+
+import (
+	"github.com/influxdata/toml"
+	"github.com/influxdata/toml/ast"
+
+	"github.com/influxdata/telegraf/migrations"
+)
+
+// Migration function to migrate deprecated RabbitMQ options
+func migrate(tbl *ast.Table) ([]byte, string, error) {
+	// Decode the old data structure
+	var plugin map[string]interface{}
+	if err := toml.UnmarshalTable(tbl, &plugin); err != nil {
+		return nil, "", err
+	}
+
+	// Check for deprecated options and migrate them
+	var applied bool
+	var message string
+
+	// Remove a deprecated name option (it should be migrated to global tags, not plugin tags)
+	// The name option is deprecated in favor of using Telegraf's global tags configuration
+	if _, found := plugin["name"]; found {
+		applied = true
+		// Remove the deprecated setting - users should use global tags instead
+		delete(plugin, "name")
+		message = "removed deprecated 'name' option; use global 'tags' configuration instead"
+	}
```

Comment:
I think we should use the plugin tags... At least I don't see anything preventing this...